src/xcode/ENA/HealthCertificateToolkit/Tests/HealthCertificateToolkitTests/DigitalGreenCertificateAccessTests.swift

```diff
@@ -9,7 +9,7 @@ import SwiftCBOR
 
 final class DigitalGreenCertificateAccessTests: XCTestCase {
 
-    func test_When_decodeSucceeds_Then_CorrectDataIsReturned() throws {
+    func test_When_decodeSucceeds_Then_CorrectCertificateIsReturned() {
```

Comment:
Very small thing: Just wondering why D in decode is small while using "Start case"

Reply:
Fixed typo.